ote/resources/public/language/sv.edn

```diff
@@ -739,54 +739,54 @@ Man måste vara inloggad för att digitalisera eller publicera mobilitetstjänst
+            :ote.db.transit/attachments "Bifogade filer"
             :ote.db.transport-operator/transport-operator "Tjänsteproducent"}
 
-  :pre-notice-dialog {:label "Muutosilmoituksen tiedot"
-                      :comments-label "Viranomaisen kommentit (näkyvät vain viranomaisille)"
-                      :new-comment-label "Kirjoita kommentti"}
+  :pre-notice-dialog {:label "Uppgifter i ändringsmeddelandet"
+                      :comments-label "Myndighetens kommentarer (visas endast för myndigheter)"
+                      :new-comment-label "Skriv en kommentar"}
 
-  :delete-pre-notice-dialog {:label "Poista luonnostilassa oleva muutosilmoitus?"
-                             :content "Haluatko poistaa tämän muutosilmoituksen pysyvästi?"
-                             :success-message "Muutosilmoitus poistettu"}
+  :delete-pre-notice-dialog {:label "Ta bort ett ändringsmeddelande med utkast-status?"
+                             :content "Vill du ta bort det här ändringsmeddelandet för gott?"
+                             :success-message "Ändringsmeddelandet har tagits bort"}
   }
 
 :pre-notice-page
-{:pre-notice-form-title "Säännöllisen henkilöliikenteen muutosilmoitus"
- :notice-type-title "Ilmoitettavan muutoksen tyyppi"
- :effective-dates-title "Muutoksen alkamispäivä tai -päivät"
- :attachment-section-title "Muutoksen tarkemmat tiedot"
- :route-and-area-information-title "Reitin ja alueen tiedot"
- :effective-date-from "Alkamispäivä"
- :effective-date-description "Selitys"
- :publish-missing-required "Du kan ännu inte publicera rutten, eftersom du inte har fyllt i alla obligatoriska uppgifter på blanketten. Fyll i de ställen som är markerade med texten \"Obligatorisk uppgift\" eller \"Uppgifter saknas\"."
- :save-success-send "Ilmoituksen lähettäminen onnistui."
- :save-success "Luonnoksen tallentaminen onnistui."
- :save-failure "Luonnoksen tallentaminen epäonnistui."
- :save-failure-send "Ilmoituksen lähettäminen epäonnistui."
- :route-description-hint "esim. Tampere - Pori tai Oulu - Seinäjoki"
- :add-attachment "Uusi liite"
- :select-attachment "Lisää liite"
- :effective-date-descriptions {:year-start "Vuoden alku"
-                               :school-start "Kouluvuoden alku"
-                               :school-end "Kouluvuoden päätös"
-                               :season-schedule-change "Aikataulukauden vaihtuminen"}
+{:pre-notice-form-title "Meddelande om ändring i regelbunden persontrafik"
+ :notice-type-title "Typ av ändring som meddelas"
+ :effective-dates-title "Ändringens begynnelsedatum"
+ :attachment-section-title "Noggrannare uppgifter om ändringen"
+ :route-and-area-information-title "Uppgifter om rutten och området"
+ :effective-date-from "Begynnelsedatum"
+ :effective-date-description "Förklaring"
+ :publish-missing-required "Du kan inte skicka meddelandet ännu, eftersom det ännu saknas obligatoriska uppgifter i blanketten. Fyll i de ställen där det står \\\"Uppgift krävs\\\"."
```

Comment:
Why there is three slashes around `\\\"Uppgift krävs\\\"`. One is enough. `\"Upgift krävs\"`